lib/rb_shift/client.rb

```diff
@@ -29,7 +29,6 @@ def initialize(url, bearer_token = nil, username = nil, password = nil)
                                              headers:    { Authorization: "Bearer #{bearer_token}" }
     end
 
-    # rubocop:disable Metrics/AbcSize
```

Comment:
My version of rubocop unfortunately complains that the ABC complexity of ```get``` method is too high. I will fix the rubocop version in separate pull request.  Lets not mix things  together.